src/main/scala/com/thenewmotion/chargenetwork/ocpp/charger/OcppCharger.scala

```diff
@@ -25,8 +27,10 @@ class OcppSoapCharger(chargerId: String,
 
 class OcppJsonCharger(chargerId: String,
                        numConnectors: Int,
-                       centralSystemUri: URI) extends OcppCharger {
-  val client: CentralSystem = new JsonCentralSystemClient(chargerId, centralSystemUri)
+                       centralSystemUri: URI,
+                       authPassword: Option[String])(
+                       implicit sslContext: SSLContext = SSLContext.getDefault)
+                       extends OcppCharger {
+  val client: CentralSystem = new JsonCentralSystemClient(chargerId, centralSystemUri, authPassword)
```

Comment:
The ocpp_charger was always using JsonCentralSystemClient. Updating that is out of the scope of this story